pynng/_aio.py

```diff
@@ -145,7 +145,10 @@ def __init__(self, obj, async_backend):
         if async_backend is None:
             async_backend = sniffio.current_async_library()
         if async_backend not in self._aio_helper_map:
-            raise ValueError('The async backend {} is not currently supported.'.format(async_backend))
+            raise ValueError(
```

Comment:
If this is too nit-picky I can just drop it.
I tend to do this kind of thing mechanically when the editor complains...